x-pack/plugins/maps/public/components/layer_addpanel/view.js

```diff
@@ -39,14 +37,21 @@ export class AddLayerPanel extends Component {
       return;
     }
 
+    const layerOptions = {};
```

Comment:
nit, but I'd write this as one-liner without the mutation

`const layerOptions = this.state.layer ? { style: this.state.layer.getCurrentStyle().getDescriptor() } : {};`

It makes the code less branch-y.